tikv-client/src/main/java/com/pingcap/tikv/meta/TiIndexInfo.java

```diff
@@ -77,18 +77,26 @@ public TiIndexInfo(
     this.isInvisible = isInvisible;
   }
 
-  public static TiIndexInfo generateFakePrimaryKeyIndex(TiTableInfo table) {
-    TiColumnInfo pkColumn = table.getPKIsHandleColumn();
-    if (pkColumn != null) {
+  public static TiIndexInfo genClusterIndex(TiTableInfo table) {
```

Comment:
introudce why we need genClusterIndex in comment. I think it is for reuse of indexscan?

Reply:
add comment